decentraland/unity-explorer
# Fix: replaced all profile pictures with default one

## What does this PR change?

Fix #869 
replaces the chat and profile widget profile pic with the default one

...

## How to test the changes?



1. Launch the explorer
2. Check that the profile picture in chat messages is the default one

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
